docs/source/user-guide/tutorials/build-pkgs-skeleton.rst

```diff
@@ -213,14 +212,14 @@ Windows:
 
 .. code-block:: text
 
-    conda convert -f --platform all C:\Users\jsmith\Miniconda\conda-bld\win-64\pyinstrument-0.13.1-py27_0.tar.bz2
+    conda convert -f --platform all C:\Users\jsmith\Miniconda\conda-bld\win-64\click-6.7-py27_0.tar.bz2
```

Comment:
Is this still a thing that people use/should use?

Reply:
conda-convert?  It's mostly been superseded by proper noarch support.  A few people still use it, especially when creating constructor installers, because we don't support noarch there yet.